tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = py34, lint
+
+[testenv]
+setenv =
+    PYTHONPATH = {toxinidir}:{toxinidir}/homeassistant
+commands = py.test {posargs}
+     flake8 --exclude www_static homeassistant
+; If you want to make tox run the tests with the same versions, create a
+; requirements.txt with the pinned versions and uncomment the following lines:
+deps =
+     -r{toxinidir}/requirements_all.txt
+     -r{toxinidir}/requirements_test.txt
+
+[testenv:lint]
+basepython = python3
+commands =
+     pylint homeassistant
```

Comment:
We should probably switch this to `script/lint` so we don't have to maintain the lint commands in two places.